### DISCUSSION TITLE
Redirect to another ip for domain name

### DISCUSSION THREAD

**Discussion by mahmutduzen** [okd-project/okd#1173] [Q&A]
I am using opensihft okd version 4.7. I have a mail server on 192.168.1.x. I directed my http and https traffic directly to 192.168.1.210 where openshift is installed. Is there a structure in Openshift where I can forward the request to mail.sitename.com to my mail server? Thanks.

**Comment by rvanderp3** [comment]
Hi @mahmutduzen you should be able to use [DNS forwarding](https://docs.okd.io/4.7/networking/dns-operator.html#nw-dns-forward_dns-operator) for this.

**Reply by mahmutduzen** [reply]
With dns fowarding, I think we can redirect directly to sitename.com. Is there a structure where I can only forward http requests to mail.sitename.com?
